src/fonts/bravura_metrics.js

```diff
@@ -317,13 +317,13 @@ export const BravuraMetrics = {
         superSubRatio: 0.66
       },
       csymDiminished: {
-        scale: 1,
+        scale: 0.8,
         leftSideBearing: -32,
         advanceWidth: 506,
         yOffset: 0
       },
       csymHalfDiminished: {
-        scale: 1,
+        scale: 0.8,
```

Comment:
Are these scale changes intentional?

Reply:
Yes, sorry I forgot to mention that.  I tweaked the scale for some symbols that were oversized vs. the text.